src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -48,4 +48,36 @@ trait DataAccess
       workflowFailureQuery.schema ++
       workflowMessageQuery.schema ++
       workflowAuditStatusQuery.schema
+
+  def truncateAll: WriteAction[Int] = {
+    // important to keep the right order for referential integrity !
```

Comment:
Update to the "new plan":
1. Truncate instead of drop, if tables exist
2. Create tables, if tables don't exist